browser/ui/webui/brave_rewards_internals_ui.cc

```diff
@@ -427,8 +427,7 @@ void RewardsInternalsDOMHandler::GetEventLogs(const base::ListValue* args) {
           weak_ptr_factory_.GetWeakPtr()));
 }
 
-void RewardsInternalsDOMHandler::OnGetEventLogs(
-    const std::vector<brave_rewards::EventLog>& logs) {
+void RewardsInternalsDOMHandler::OnGetEventLogs(ledger::EventLogs logs) {
```

Comment:
Should still be `const ledger::EventLogs&`.

Reply:
fixed